src/server/routes/revision.js

```diff
@@ -168,17 +168,27 @@ function diffRevisionsWithParents(orm, entityRevisions, entityType) {
 		(revision) =>
 			revision.parent()
 				.then(
-					(parent) => makePromiseFromObject({
-						changes: revision.diff(parent),
-						entity: revision.related('entity'),
-						entityAlias: revision.get('dataId') ?
-							revision.related('data').fetch({require: false, withRelated: ['aliasSet.defaultAlias', 'aliasSet.aliases']}) :
-							orm.func.entity.getEntityParentAlias(
-								orm, entityType, revision.get('bbid')
-							),
-						isNew: !parent,
-						revision
-					}),
+					(parent) => {
+						const dataId = revision.get('dataId');
+						let isNew = false;
+						let isDeletion = false;
+						if (!parent) {
+							isNew = dataId;
```

Comment:
```suggestion
							isNew = Boolean(dataId);
```

I guess this is a bit clearer and makes sure we use a boolean.